src/MaxBTCCore.sol

```diff
@@ -557,8 +567,7 @@ contract MaxBTCCore is Initializable, UUPSUpgradeable, Ownable2StepUpgradeable {
         }
 
         Batch storage batch = _activeBatch();
-        batch.maxBtcBurned += maxBtcAmount;
-        MaxBTCERC20(config.maxBtcToken).burn(_msgSender(), maxBtcAmount);
```

Comment:
Since we are neither burning nor withdrawing maxBTC from user balance here, there are now several issues I see:
- user can request to withdraw more maxBTC than they have
- user can keep infinitely requesting to withdraw maxBTC in a loop